ui/src/components/layout/AuthState.js

```diff
@@ -1,64 +1,29 @@
 import React from 'react';
```

Comment:
This has nothing to do with validation jobs.
Additionally removing the ability to login or logout at any time has no purpose